apps/blockchain/lib/blockchain/transaction.ex

```diff
@@ -255,10 +266,18 @@ defmodule Blockchain.Transaction do
 
       # Note, we only want to take the first 3 items from the tuples,
```

Comment:
Is this still true now that we're taking all four items?

Reply:
Updated this in 0ff7ee6ae5abc3a06d32cd9d9d43e36f24ec8fa7